src/main/java/org/arl/fjage/remote/ConnectionHandler.java

```diff
@@ -70,6 +83,7 @@ public void run() {
         println(ALIVE);
       }
     }
+    fw.authenticate(conn, null);
```

Comment:
What does this check on the connection do here? Also `fw.authenticate` returns a true/false. So is the idea that a firewall that doesn't allow this connection will close it inside it's `authenticate` implementation?